content/docs/contributing/to-docs/style-guide/format.md

```diff
@@ -195,129 +127,88 @@ Syntax highlighting in the code block should conform to the [contrast guidelines
 
 When writing code blocks, make sure to include a language identifier. For C++ code blocks, use the `cpp` identifier.
 
-### Mark placeholders with angle brackets
 
-Use angle brackets for placeholders. Use the text within the brackets to tell the reader what a placeholder represents. For example:
+### Links
 
-```shell
-git push origin <your-branch-name>
-```
-
-### Bold user interface elements
-
-Do | Don't
-:--| :-----
-Choose **Edit**. | Click "Edit".
-.. the **Play** button. | ... the Play button.
-
-### Bold input combinations
-
-Do | Don't
-:--| :-----
-Press **Enter**. | Press "Enter".
-.. **right-click** the asset name ... | ... right-click the asset name ...
-Hold **CTRL+SHIFT** ... | Hold `Control + Shift` ...
-
-### Bold proper application names
-
-For the first introduction of an application name on a page, use **bold** text. Subsequent references to the application on the page should not be bold.
-
-Example:
-
-**O3DE Editor** is the primary development environment for .... Open O3DE Editor by launching it from...
+Write links without the file extension *.md*.
 
-Do | Don't
-:--| :-----
-Open **Material Editor**. | Open the Material Editor.
-In **Asset Processor**, right-click... | In "Asset Processor", right-click...
+| Relative link | Result |
```

Comment:
What about absolute links? Do you want to show some examples of those here too?

Reply:
Added. Also added guidance on external links.